volume_scroller@francislavoie.github.io/extension.js

```diff
@@ -62,13 +62,22 @@ export default class VolumeScrollerExtension extends Extension {
   _handle_scroll(_actor, event) {
     let volume = this.sink.volume;
 
-    switch (event.get_scroll_direction()) {
+    let settings = new Gio.Settings({
+      schema_id: "org.gnome.desktop.peripherals.touchpad",
+    });
+    
+    let naturalScroll = settings.get_boolean("natural-scroll");
+    let multiplier = naturalScroll ? -1 : 1;
+
+    let scrollDirection = event.get_scroll_direction();
+
+    switch (scrollDirection) {
```

Comment:
Oh and this; can use const here since these values don't change and are just settings collection:

```suggestion
    const settings = new Gio.Settings({
      schema_id: "org.gnome.desktop.peripherals.touchpad",
    });
    
    const naturalScroll = settings.get_boolean("natural-scroll");
    const multiplier = naturalScroll ? -1 : 1;

    const scrollDirection = event.get_scroll_direction();
    switch (scrollDirection) {
```

Reply:
Thanks, I am a beginner so I can miss things like this :)

Reply:
No problem at all, that's what review is for! 😊